raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -92,6 +103,23 @@ def spawn_create_channel(self, partner_address: Address) -> None:
         )
         self._spawn_web_rtc_coroutine(coroutine, self._handle_sdp_callback, partner_address)
 
+    def set_candidates(
+        self, partner_address: Address, description: Dict[str, Union[str, Sequence[str]]]
+    ) -> None:
+        assert self.node_address, "Transport is not started yet but tried to set candidates"
+        rtc_partner = self.get_rtc_partner(partner_address)
+        msg = "Partner RTCPeerConnection is not set, but tried to set candidates"
+        assert rtc_partner.peer_connection is not None, msg
```

Comment:
This assert will never throw. By calling `get_rtc_partner` a new partner is created whenever there is no existing one. A RTC Partner always has a `RTCPeerConnection` object set. If you want to explicitly check if there is already an existing `rtc_partner` object you can do it by `partner_address in self.address_to_rtc_partner`.

I think it makes sense to ask if a rtc_partner already exists because it must exist in order to receive candidates.